AuthorYYMMDD_fignr_001.yaml

```diff
@@ -0,0 +1,74 @@
+# This file can exist with the "source" information only, the entry data exists should be marked as "False" 
+# to indcate that neither the metadata nor the actual data were digitized or provided.
+# use "null" to idicate that a value is not available in the literature
+version: 1 # Is this necessary. Isn't this anyway given by git versioning
+
+source: 
+    doi: https://doi.org/10.1016/0039-6028(85)90985-9
+    bib id: Beden1985 # First Author and YYYY. Cross check for duplicats in the bib file.

```

Comment:
I think I would shorten some things here to make them more consistent:

`bib id` is inconsistent with `curve identifier`. Maybe it's better to just use `bib` or `bibliography` instead.

Reply:
If it does not necessarily have to imply that it refers to a BibTeX file `bibliography` is fine. 
Or maybe `bibtexlabel`

Reply:
```suggestion
    bib: Beden1985 # First Author and YYYY. Cross check for duplicats in the bib file.

```